pkg/manifests/manifests_test.go

```diff
@@ -851,16 +851,46 @@ func TestPrometheusOperatorConfiguration(t *testing.T) {
 func getContainerArgValue(containers []v1.Container, argFlag string, containerName string) string {
 	for _, container := range containers {
 		if container.Name == containerName {
-			for _, arg := range container.Args {
-				if strings.HasPrefix(arg, argFlag) {
-					return arg
-				}
+			arg, ok := getArgValue(container, argFlag)
+			if ok {
+				return arg.original
 			}
 		}
 	}
 	return ""
 }
 
+type argValue struct {
+	key      string
+	value    string
+	original string
+}
+
+// returns the value of the container arg if found or false
+func getArgValue(container v1.Container, flag string) (argValue, bool) {
```

Comment:
I think JP added a similar test helper in the past. Is it possible to reuse or repurpose that one?

Reply:
that is the function `getContainerArgValue` above which is now using this, wdyt?

Reply:
Makes sense 👍